src/main/webapp/dev/js/common/instructorFeedbackResults.es6

```diff
@@ -308,13 +330,17 @@ function prepareInstructorFeedbackResultsPage() {
                 displayAjaxRetryMessageForPanelHeading(displayIcon);
             },
             success(data) {
+                const remindButtonContent = $(data).find('.remind-no-response')[0];
                 $(panelCollapse[0]).html(getAppendedResponseRateData(data));
                 $(panelHeading).removeClass('ajax-response-submit');
                 $(panelHeading).removeClass('ajax-response-auto');
                 $(panelHeading).off('click');
-                displayIcon.html('<span class="glyphicon glyphicon-chevron-down pull-right"></span>');
-                $(panelHeading).click(toggleSingleCollapse);
+                displayIcon.html(remindButtonContent);
+                displayIcon.append('<span class="glyphicon glyphicon-chevron-down pull-right"></span>');
+                $(panelHeading).data('state', 'up');
+                $(panelHeading).click(toggleNoResponsePanel);
                 $(panelHeading).trigger('click');
```

Comment:
I might have missed this earlier, but $(panelHeading) should be extracted to a variable.